contracts/exchange/test/core.ts

```diff
@@ -56,6 +55,8 @@ chaiSetup.configure();
 const expect = chai.expect;
 const blockchainLifecycle = new BlockchainLifecycle(web3Wrapper);
 
+import { FillOrderWrapper } from './assertion_wrappers/fill_order_wrapper';
+
 // tslint:disable:no-unnecessary-type-assertion
 describe('Exchange core', () => {
```

Comment:
Can we upgrade these tests to use the new `blockchainTest()` and import semantics while we're here? Should shave off a whole ~10 lines!

Reply:
Darn tootin we can ! #RIP